spec/shr_oncology.txt

```diff
@@ -82,31 +83,267 @@ includes 0..1	BreastCancerHistologicGrade
 		Concept:		SCT#260869008 "Timing of stage"
 		Description:	"Indicates when the staging was done, in terms of treatment landmarks."
 		Value:			CodeableConcept	from StageTimingPrefixVS
-		
-		Element:		BreastCancerPrimaryTumorClassification
-		Concept:		SCT#78873005 "T category"
-		Based on:		CodedEvaluationComponent
-		Description:	"The size and extent of the primary tumor, based on criteria defined by the staging system being used."
-		Value:			CodeableConcept
-		0..1			StageSuffix
 
-		Element:		BreastCancerRegionalNodesClassification
-		Concept:		SCT#277206009 "N category"	//RMB: was LNC#59525-6, which is specific to AJCC 7th edition
-		Based on:		CodedEvaluationComponent
-		Description:	"The presence of metastases in regional lymph nodes, based on criteria defined by the staging system being used."
-		Value:			CodeableConcept
-		0..1			StageSuffix
-
-				Element:	StageSuffix
-				Concept:	TBD	
-				Description: "A suffix used in conjuction with certain breast cancer stages, based on criteria defined by the staging system being used."
-				Value: 		CodeableConcept
-				
-		Element:		BreastCancerDistantMetastasesClassification
-		Concept:		SCT#277208005 "M category (observable entity)"	//RMB: was LNC#59522-3, which is specific to AJCC 7th edition
-		Based on:		CodedEvaluationComponent
-		Description:	"The presence of distant metastases, based on criteria defined by the staging system being used."
-		Value:			CodeableConcept
+
+	EntryElement:	BreastCancerPrimaryTumorClassification
+	Based on:		CodedLaboratoryObservation
+	Concept:		SCT#78873005 "T category"
+	Description:	"The size and extent of the primary tumor, based on criteria defined by the staging system being used."
+					ObservationContext.ResultValue.CodeableConcept from TBD
+					LaboratoryObservationTopic.Focus value is type Identifier  // Lymph node identifier
+					LaboratoryObservationTopic.TopicCode is SCT#78873005 "T category"
+	0..0			LaboratoryObservationTopic.DiagnosticService  
+	0..0			LaboratoryObservationTopic.Specimen // also removed b/c this is determined lower
+	0..0			LaboratoryObservationTopic.Precondition
+	0..0			LaboratoryObservationTopic.FindingMethod
+			 		LaboratoryObservationTopic.EvaluationComponent
+	includes 0..1	StageSuffix
+					LaboratoryObservationTopic.PanelMembers.Observation
```

Comment:
@markkramerus - TumorAssessment now removed from BreastCancerPrimaryTumorClassification as the intermediate level.